### PR TITLE
[Term Entry] Lua Math .sqrt()

### PR DESCRIPTION
**Pull request by deenovita**
### Description
- Created a new term entry. Lua math sqrt() function.
- Close the issue #3506 

### Type of Change

- Added a new entry.

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/lua/concepts/mathematical-library/terms/sqrt/sqrt.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+```lua
+num_var = -25
+math.sqrt(num_var)
+```
+
+The results will look like this:
+
+```shell
+nan
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/sqrt/sqrt.md:56]:
```suggestion
-nan
```

File: content/lua/concepts/mathematical-library/terms/sqrt/sqrt.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+The result will be shown below:
+
+```shell
+5.0
+```
+
+## Example 2
+
+The function returns nan as the results value if the `num_var` has negative number.
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/sqrt/sqrt.md:46]:
```suggestion
The code below demonstrates the method with a negative value.
```

File: content/lua/concepts/mathematical-library/terms/sqrt/sqrt.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+math.sqrt(num_var)
+```
+
+The results will look like this:
+
+```shell
+nan
+```
+
+Where nan stands for not a number which represent undefined or unpresentable numerical results.
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/sqrt/sqrt.md:59]:
```suggestion
Where `nan` stands for "not a number", which represent undefined or unpresentable numerical results.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@deenovita I've added my initial comments. Please update the entry, and tag me when you've finished making changes. Thanks

File: content/lua/concepts/mathematical-library/terms/sqrt/sqrt.md (hunk 1 above)
Comment (deenovita) [reply content/lua/concepts/mathematical-library/terms/sqrt/sqrt.md:56]: I'm not sure if I should change `nan' to `-nan', the result I get is `nan'
lua-nan

**Comment by deenovita** [comment]
@caupolicandiaz I've finished making the changes. There's one I'm not sure of changing since the result I get is still the same `nan` not `-nan`. Let me know if there is another change I need to make, thank you

**Review by caupolicandiaz — APPROVED** [review]
@deenovita This is ready for a second review

File: content/lua/concepts/mathematical-library/terms/sqrt/sqrt.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  - 'Functions'
+  - 'Math'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **math.sqrt()** function returns the result of the square root operation on a numeric variable. It takes a positive number as an argument value to be passed into the function.
+
```
Comment (KTom101) [inline content/lua/concepts/mathematical-library/terms/sqrt/sqrt.md:18]:
```suggestion

> **Note:** Results are dependent on the compiler. There may be a small variation or results may not be as precise as expected.

```

File: content/lua/concepts/mathematical-library/terms/sqrt/sqrt.md (hunk 1 above)
Comment (KTom101) [reply content/lua/concepts/mathematical-library/terms/sqrt/sqrt.md:56]: Perhaps let's just add a note on this in the main body. This is most likely due to the compiler. I tried two different compilers and got both answers. I added a blockquote note for this in the intro.

**Review by KTom101 — CHANGES_REQUESTED** [review]
@deenovita I've added a change. Please let me know when you're done.

**Comment by deenovita** [comment]
@caupolicandiaz @KTom101 Thanks for the reviews, I tried to use Lua ver 3.54 and 3.53 on Windows, and the results were the same `nan`. When I try to use the online compiler it shows `-nan` which is interesting.
I have added a blockquote note as suggested to avoid confusion. Thank you very much!

**Review by KTom101 — APPROVED** [review]
@deenovita Looks good, I will be merging soon.

**Pull request merged**